website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+After some investigation, we realized that a couple issues prevented the community from taking action on the deprecation warnings. First, the deprecation warning was not always actionable which caused people to ignore them ([issue one](https://facebook/react-native#34650), [issue two](https://github.com/react-native-community/cli#1699)). Second, the deprecation warnings were being [incorrectly filtered](https://facebook/react-native@fa2842d) by `LogBox.ignoreLogs`. Both of these have now been fixed, but we want to give people more time to upgrade the deprecated call sites.
+
+So in this release we are adding back React Native’s prop types so that it is easier for people to upgrade and migrate their code to avoid using them. The `deprecated-react-native-prop-types` package has also been updated for all of the props in 0.71. In the future, we plan to proceed with the deprecation and remove prop types once again. We expect that when we revisit the removal, the community will experience significantly less issues.
+
+:::caution
+
+As part of this change, we are also removing the console filtering from `LogBox.ignoreLog`. This means logs that you have previously filtered with `Logbox.ignoreLog` will start apearing again in the console when you upgrade.
+
+This is expected, so that since warnings and errors such as deprecation warnings can be found and fixed.
+:::
+
+## Developer Experience Improvements
+
+### React DevTools
+
+In this release, we've brought two popular React DevTools features on web to React Native.
+
+Click to inspect is the option in the top left corner of React Dev Tools that allows you to click on an item in the app to inspect it in Dev Tools, similar to the Chrome element inspector.
+
+Component highlighting will highlight the element you select in DevTools in the app so you can see which React components line up with which on-screen elements.
+
+Here are both features in action:
+
+![Video of the behavior described above in action. On the left is a React Native app running in an iPhone simulator. On the right is the React DevTools. In both workflows, clicking on an item in the DevTools highlights the corresponding components in the app.](/blog/assets/ElementInspecting.gif)
+
+### Hermes
+
+In React Native 0.70, we made [Hermes the default engine for React Native](https://reactnative.dev/blog/2022/07/08/hermes-as-the-default).
+
+In React Native 0.71, we’re upgrading Hermes with a few notable improvements:
+
+- **Improve source maps**: By loading source maps over the network with Metro we’ve restored the ability to use source maps in recent versions of Chrome Dev Tools outside of Flipper.
+- **Improve `JSON.parse` performance**: This version includes a performance optimization that improves the performance of `JSON.parse` up to 30%.
+- **Add support for `.at()`**: Hermes now [_includes support requested_](https://github.com/facebook/hermes/issues/823) for the `.at()` method for `String`, `TypedArray`, and `Array`.
```

Comment:
This reads as though `at` was already support but just missing these features. Is that correct? I can't tell from the linked post.

If not, could this just be:

Hermes now supports `.at()` on String, TypedArray, and Array.